README.md

```diff
@@ -123,31 +135,10 @@ Path:       /home/linuxbrew/.linuxbrew/Cellar/python@3.12/3.12.5
 Executable: /home/linuxbrew/.linuxbrew/Cellar/python@3.12/3.12.5/bin/python3.12
 ```
 
-Now this virtual environment needs to be removed, because the path to this folder vary from machine and we need debugger to access these virtual environment in more consistent way. 
-
-
-Remove the virtual env by running:
-> Note: If you don't see any path listed under Virtualenv, you can skip this step.
+Create a settings.json file to point the POETRY_ENV_INFO to the Executable directory in the Virtualenv
+```json
+{"POETRY_ENV_INFO": "/home/mitchell/ComputerScience_WSL/Incredihire/Survey/backend/.venv/bin/python"}
```

Comment:
I would revise the debugging section to:
1. Explain how to select poetry venv as the python interpreter from VSCode
2. Remove steps for configuring poetry to create venv within the project, since we don't need to explicitly specify the path from `launch.json`

Reply:
Done.  Thanks for figuring this out.